modules/vulnerability-management20-report-review-create.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-report-review-create_{context}"]
+= Reviewing report configuration and creating
+
+[role="_abstract"]
+You can review the configuration of your vulnerability report before creating it.
```

Comment:
You can review the details of your vulnerability report configuration before creating it.